### ISSUE TITLE
Handle colors

### ISSUE THREAD

**Issue by kj7rrv** [CatimaLoyalty/catima.py#1]
This needs to handle colors when creating `Card`s. Right now they are just ignored.

**Comment by kj7rrv** [comment]
@theLastProject is it  okay to ignore the color's alpha component?

**Comment by TheLastProject** [comment]
I didn't even know there was an alpha component.

I think I kinda made a bad design decision letting those be Android colours though, I should have probably gone with hex instead.

How hard is it to parse Android colours? Should I perhaps consider breaking backwards compatibility and just start putting hex values in the URL instead?

**Comment by kj7rrv** [comment]
Android colors shouldn't be too hard to parse. Breaking backwards compatibility shouldn't be too big of an issue, because color isn't really critical. Another option would be to prefix hex colors with an `x`, so when importing old data, it would still accept it as an Android color int, but new data would be hex. It still wouldn't solve the problem of old versions importing new data, though... Maybe it would be best to just leave it as it is. I should be able to parse the Android colors just fine; it looks like a simple encoding.

73

**Comment by kj7rrv** [comment]
Supporting both would just make the decoder more complicated, so it's probably best to stick with Android colors for now, and maybe switch to hex in a future version if any other breaking changes are made.
